contrib/examples/apiserver/broker.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   # service-catalog will use the contents of the secret. The secret should have "username"
   # and "password" keys
   authInfo:
-    basicAuthSecret:
+    secretRef:
```

Comment:
@nilebox would it be possible to eliminate the need to use `secretType`? When I put in the code for `authInfo`, I allowed for other keys besides `basicAuthSecret`. For example:

```yaml
authInfo:
    bearerTokenAuthSecret:
        namespace: some-namespace
        name: secret-name
```

Either way is fine with me, I'm just looking for an explanation for why the change.